### ISSUE TITLE
Feature request: emit a log entry when an on-done script completes

### ISSUE THREAD

**Issue by TerrorJack** [transmission/transmission#5395]
### What is the issue?

I've set up an on-done script for `transmission-daemon`. When a torrent finishes downloading, `journalctl -u transmission.service | grep script` will show a `Calling script` log entry which indicates the script started execution. It would be nice if there's another log entry containing that script's exit status when it finishes execution, otherwise there's no way to know if it crashed silently after some time.

### Which application of Transmission?

transmission-daemon

### Which version of Transmission?

transmission-daemon 3.00 (0)

**Comment by woojh3690** [comment]
I agree with this point as well. It's difficult to know if the script has ended properly once it's started, so it's somewhat hard to determine whether the function I wanted has executed correctly due to script bugs, or if there are no issues with the script.

Additionally, although it might be challenging, I wish the standard output of the script could also be included in the transmission logs. Currently, I'm running the transmission-daemon in a docker container, and it would be really awesome if I could see the script's standard output with the 'docker logs -f' option.
